srcpkgs/QTAlarm/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'QTAlarm'
+pkgname=QTAlarm
+version=2.3.2
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-multimedia-devel qt5-core qt5-gui qt5-widgets qt5-declarative-devel"
```

Comment:
`-devel` packages usually belong in `makedepends=`

Reply:
Then dependencies in `depends=` are resolved automatically

Reply:
Thanks. It seems I was testing it by trying to put that into "make_depends" instead of "makedepends". I applied these suggestions to the most recent commit.